lib/datadog.ts

```diff
@@ -62,4 +72,30 @@ export class Datadog extends cdk.Construct {
       this.transport.setEnvVars(lambdaFunctions);
     }
   }
+
+  public validateProps(props: DatadogProps) {
```

Comment:
Do you think validateProps should be a public method on the construct class, or should it just be a standalone function? I would lean towards making it a standalone function, just so it's easier to test. Also, I doubt it should be public, not sure why other code would need to call this.